syntax = "proto2";

package oeo;

message ComputerHealth {
    required uint32 id = 1;
    required ComputerHealthData data = 2;
}

message ComputerHealthData {
    required uint32 heartbeat = 1;
    required bool vehicle_script_running = 2;
    required bool radio_script_running = 3;
    required bool traffic_script_running = 4;
    required float cpu_utilization = 5;
    required float memory_utilization = 6;
}